### PR TITLE
Docs for Saved Searches for the Aug release

### PR DESCRIPTION
**Pull request by MaedahBatool**
This PR updates the Saved Searches docs. 

- https://linear.app/sourcegraph/issue/SRCH-786/update-saved-searches-docs

### CODE REVIEW

**Comment by MaedahBatool** [comment]
@sqs can I get a review here 🙌

File: docs/code-search/working/saved_searches.mdx [hunk 1]
```diff
@@ -1,50 +1,38 @@
-# Saved Search
+# Saved Searches
 
 <p className="subtitle">This page provides docs about how Saved Searches work with Sourcegraph.</p>
 
-Saved Search lets you save and describe search queries so you can easily find and use them again later. You can create a saved search for anything, including diffs and commits across all branches of your repositories.
+Saved Searches lets you easily save and describe search queries to find and use them again later. You can create a saved search for anything, including diffs and commits across all branches of your repositories.
```
Comment (sqs) [inline docs/code-search/working/saved_searches.mdx:5]: can be worded more simply as "reuse and share"

File: docs/code-search/working/saved_searches.mdx [hunk 2]
```diff
@@ -1,50 +1,38 @@
...
-Saved searches can be associated with a user or an org:
+## Creating saved searches
 
-- User saved searches are only visible to (and editable by) the user that created them.
-- Org saved searches are visible to (and editable by) all members of the org.
+To create a new saved search:
 
-### User saved searches
+- Go to the Saved Searches section and click the **New saved search** button
+- Fill out the description field and define the syntax for the search query
```
Comment (sqs) [inline docs/code-search/working/saved_searches.mdx:16]: "Define the syntax" sounds wrong. I'd say it's "enter the search query". Define the syntax sounds like you are defining a new way of specifying search queries

File: docs/code-search/working/saved_searches.mdx [hunk 3]
```diff
@@ -1,50 +1,38 @@
...
 
-1. Go to **User menu > Saved searches** in the top navigation bar.
-1. Press the **+ Add new search** button.
-1. In the **Query** field, type in the components of the search query.
-1. In the **Description** field, type in a human-readable description for your saved search.
-1. Click **Create**. The saved search is created, and you can see the number of results.
+![create-new-saved-search](https://storage.googleapis.com/sourcegraph-assets/Docs/create-new-saved-search-0824.png)
 
-Alternatively, to create a saved search from a search you've already run:
+Enable the checkbox for **Draft** if you don't want other users to use your saved search. This is useful for testing the query syntax before sharing it with others. Once done, click the **Create saved search** button to be redirected to the Saved Searches page.
```
Comment (sqs) [inline docs/code-search/working/saved_searches.mdx:23]: Don't use "syntax" here. Just call the noun a "search" or "query"

For Draft, I'd recommend something like "... if you want other users to know that it's not ready and shouldn't be relied upon". They can still see drafts, it's just a visual indicator.

**Review by sqs — APPROVED** [review]

**Pull request merged**